src/game/server/client.cpp

```diff
@@ -600,55 +600,119 @@ void SV_CreateClientCommands()
 
 	g_ClientCommands.Create("ent_find_by_classname", [](CBasePlayer* player, const CommandArgs& args)
 		{
-			// TODO - Pagination
 			if (args.Count() > 1)
 			{
-				int count = 0;
-				ALERT(at_console, "entindex - classname - targetname - origin");
+				bool doPagination = true;
```

Comment:
You can probably refactor the pagination algorithm to reuse the common elements, like using a struct with a template function taking a lambda to handle the meat of the iteration logic.

Everything else looks good.